### PR TITLE
feat: added bootstrap.ps1 and instructions

### PR DESCRIPTION
**Pull request by Elizabeth-Freemarket**

### CODE REVIEW

File: bootstrap.ps1 [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Connect-AzAccount
+
+$serviceBusConnectionString = (Get-AzServiceBusKey -Name RootManageSharedAccessKey -NamespaceName fmfx-uk-qa-eun-shared-sbus -ResourceGroupName fmfx-uk-qa-eun-resgrp).PrimaryConnectionString
+$bankingStorageKey = (Get-AzStorageAccountKey -ResourceGroupName fmfx-uk-qa-eun-resgrp -Name fmfxukqaeunbankst | Where-Object KeyName -eq 'key2').Value 
+$cosmosKey = (Get-AzCosmosDBAccountKey -ResourceGroupName fmfx-uk-qa-eun-resgrp -Name fmfx-uk-qa-banking-cosno -Type "ReadOnlyKeys").PrimaryReadonlyMasterKey
+
```
Comment (Elizabeth-Freemarket) [inline bootstrap.ps1:6]: Unfortunately trying to login to keeper here just causes it to hang and not complete the run. But when logged in it works just fine.

File: bootstrap.ps1 [hunk 2]
```diff
@@ -0,0 +1,47 @@
+Connect-AzAccount
+
+$serviceBusConnectionString = (Get-AzServiceBusKey -Name RootManageSharedAccessKey -NamespaceName fmfx-uk-qa-eun-shared-sbus -ResourceGroupName fmfx-uk-qa-eun-resgrp).PrimaryConnectionString
+$bankingStorageKey = (Get-AzStorageAccountKey -ResourceGroupName fmfx-uk-qa-eun-resgrp -Name fmfxukqaeunbankst | Where-Object KeyName -eq 'key2').Value 
+$cosmosKey = (Get-AzCosmosDBAccountKey -ResourceGroupName fmfx-uk-qa-eun-resgrp -Name fmfx-uk-qa-banking-cosno -Type "ReadOnlyKeys").PrimaryReadonlyMasterKey
```
Comment (Elizabeth-Freemarket) [inline bootstrap.ps1:5]: This step requires PIM

**Review by akso-ak — APPROVED** [review]
LGTM

**Review by timabell-gnf1 — APPROVED** [review]

**Comment by timabell-gnf1** [comment]
I'm hoping to replace all these yaks with `az login` like we have in liquidity at some point. But thanks for documenting the setup, I'm about to half to go through it myself

**Comment by timabell-gnf1** [comment]
p.s. this is all moving to the main repo once #1 is merged

https://github.com/FreemarketFX/PlatformTests/pull/128

**Pull request merged**